sdk/keyvault/azure-security-keyvault-keys/test/ut/key_client_import_test_live.cpp

```diff
@@ -69,11 +69,4 @@ TEST_F(KeyVaultClientTest, ImportKey)
   EXPECT_EQ(key.CurveName.Value().ToString(), returnedkey.Key.CurveName.Value().ToString());
   EXPECT_EQ(returnedkey.KeyOperations().size(), 1);
   EXPECT_EQ(returnedkey.KeyOperations()[0].ToString(), KeyOperation::Sign.ToString());
-
-  {
-    // delete + purge
-    auto op = keyClient.StartDeleteKey(keyName);
-    op.PollUntilDone(std::chrono::milliseconds(1000));
-    keyClient.PurgeDeletedKey(keyName);
-  }
```

Comment:
How's the test coverage?
Should we add this and other things back, or maybe re-write them to get coverage?

Reply:
naaa,, All tests were cleaning all the keys created during the test. But this was making the tests to take longer b/c we needed to wait for the delete+purge time.  Since the key vault account is created for the test and destroyed once tests completes, we don't really need to take care about cleaning stuff after each test